mlrun/execution.py

```diff
@@ -573,22 +581,24 @@ def log_iteration_results(self, best, summary: list, task: dict, commit=False):
         """Reserved for internal use"""
 
         if best:
+            best_context = MLClientCtx.from_dict(
+                task, store_run=False, include_status=True
+            )
```

Comment:
why do you need this specifically? to get the best iteration context logged artifacts?

Reply:
yes, added a comment